### PR TITLE
Update: `JetBrains.IntelliJIDEA.Community.EAP` version 212.4746.2

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20944)

### CODE REVIEW

File: manifests/j/JetBrains/IntelliJIDEA/Community/EAP/212.4746.2/JetBrains.IntelliJIDEA.Community.EAP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+#  Scope:
```
Comment (OfficialEsco) [inline manifests/j/JetBrains/IntelliJIDEA/Community/EAP/212.4746.2/JetBrains.IntelliJIDEA.Community.EAP.installer.yaml:15]:
```suggestion
  Scope: machine
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Dude just fill in this field so we don't have to do it for every PR

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Also i don't think this is EAP
image

Or is EAP early release?

**Comment by vedantmgoyal2009** [comment]
I downloaded from the Early Access  section on JetBrains website.

**Comment by vedantmgoyal2009** [comment]
Please also tell how to decide whether to keep `Scope: machine` or `Scope: user`.

**Comment by OfficialEsco** [comment]
> Please also tell how to decide whether to keep `Scope: machine` or `Scope: user`.

If its a machine wide installer its `machine`, if it installs only for the user its `user`.
Aka Administrator access or no Administrator access required.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**